README.md

```diff
@@ -2,7 +2,10 @@
 
 # ls-lint - github action
 
-An extremely fast directory and filename linter - Bring some structure to your project filesystem
+This repository is the GitHub action that brings [ls-lint](https://github.com/loeffel-io/ls-lint) to your project easily.
```

Comment:
please swap line 5 with 7

Reply:
Sure

Reply:
I'll move the link to ls-lint on content currently line 7, then